app/src/main/java/org/wikipedia/page/PageTitle.kt

```diff
@@ -75,6 +76,12 @@ data class PageTitle(
             return mainPageTitle == displayText
         }
 
+    val isContributions: Boolean
+        get() {
+            val title = text.split('/')[0]
```

Comment:
Would it be better if we change the `[0]` to `.firstOrNull()` for null-safety purpose?

Reply:
Correct!